common/utils/src/main/resources/error/error-classes.json

```diff
@@ -860,6 +860,50 @@
       "Exceeds char/varchar type length limitation: <limit>."
     ]
   },
+  "EXPECT_PERMANENT_VIEW_NOT_TEMP" : {
+    "message" : [
+      "'<operation>' expects a permanent view but <viewName> is a temp view."
+    ]
+  },
+  "EXPECT_TABLE_NOT_VIEW" : {
+    "message" : [
+      "'<operation>' expects a table but <viewName> is a view."
+    ],
+    "subClass" : {
+      "NO_ALTERNATIVE" : {
+        "message" : [
+          ""
+        ]
+      },
+      "USE_ALTER_VIEW" : {
+        "message" : [
+          "Please use ALTER VIEW instead."
+        ]
+      }
+    }
+  },
+  "EXPECT_TABLE_OR_PERMANENT_VIEW_NOT_TEMP" : {
```

Comment:
```suggestion
  "EXPECT_PERMANENT_TABLE_OR_VIEW_NOT_TEMP_VIEW" : {
```

Reply:
in case we add temp table in the future.

Reply:
Actually, seems we can reuse `EXPECT_PERMANENT_VIEW_NOT_TEMP`? If we get a temp view, it's probably good enough to notify users that the operation needs permanent view not temp, no need to mention that the operation accepts table as well.

Reply:
That seems quite reasonable. Let me give it a try, and at the same time, let me to double check the rationality of similar cases in UT.

Reply:
I checked two cases in UT，as follows:
- "SHOW CREATE TABLE<viewName> ..."
- "ANALYZE TABLE<viewName> ..."

throw ` EXPECT_ PERMANENT_ VIEW_ NOT_ TEMP`, It's reasonable.